scripts/python/circuit.py

```diff
@@ -75,4 +79,6 @@ def count_r1cs_nonzero_terms():
             print(f"The matrix C has {c_nonzero:,} nonzero terms.")
             print("-------------------------------------------------")
             print(f"Total number of nonzero terms: {total_nonzero :,} .")
+            print(f"Union of nonzero terms count: {union_nonzero :,} .")
```

Comment:
@rex1fernando, just to clarify: this is just the # of non-zero terms in $A + B + C$ or in $r_1 A + r_2 B + r_3 C$, where the $r_i$'s are random.

So our potential sparse MLE will have this many non-zero entries, right?